tests/letstest/multitester.py

```diff
@@ -99,11 +99,9 @@
 # Globals
 #-------------------------------------------------------------------------------
 BOULDER_AMI = 'ami-072a9534772bec854' # premade shared boulder AMI 18.04LTS us-east-1
-LOGDIR = "" #points to logging / working directory
-# boto3/AWS api globals
-AWS_SESSION = None
-EC2 = None
+LOGDIR = "letest-%d"%int(time.time()) #points to logging / working directory
```

Comment:
Here so we have a dedicated LOGDIR for each execution of multitester, am I right ?

Since the name is non-predictable anymore, could we print at the beginning of the multitester execution the LOGDIR value, to help find the appropriate folder for this given execution during troubleshooting?

I have no particular opinion on the implementation, but I would personally replace `LOGDIR` by a function that generate the name and print its value, and then this function would be call appropriately at the beginning of multitester execution.

Reply:
> Here so we have a dedicated LOGDIR for each execution of multitester, am I right ?

Yep!

> Since the name is non-predictable anymore, could we print at the beginning of the multitester execution the LOGDIR value, to help find the appropriate folder for this given execution during troubleshooting?

If I understand you correctly, this is done at https://github.com/certbot/certbot/pull/7057/files#diff-821c98fdcbcf065345fc89980bc6609eR410 which is the first use of `LOGDIR` when `main` is called.

Reply:
Yes indeed, I missed it!

Reply:
No worries! It's pretty hard to see because the diff shown here is so bad.